### PR TITLE
Create en.md for Changing Shell page

### PR DESCRIPTION
**Pull request by Bridouz**
Moving from Bash to an other shell could be tricky if you don't understand how _chsh_ works.

This page tries to give informations about shells and how to use _chsh_.

### CODE REVIEW

File: configuration/changing-shell/en.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
++++
+title = "Changing Shell"
++++
+# Changing Shell
+
+A Unix shell is a command-line interpreter or shell that provides a traditional user interface for the Unix operating system and for Unix-like systems.
+Users direct the operation of the computer by entering commands as text for a command line interpreter to execute or by creating text scripts of one or more such commands.
```
Comment (JoshStrobl) [inline configuration/changing-shell/en.md]: First two paragraphs are blatant copy / paste from Wikipedia. If we could re-word that, that'd be great.

File: configuration/changing-shell/en.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+A Unix shell is a command-line interpreter that provides an interface and functionalities for your terminal. The most known and used shell in Linux is *bash* as it comes installed and activated on most distribution out-of-the-box.
+
+Other shells exist, such as :
+  - Dash
+  - Fish
+  - Zsh
+  
+A list of available shells on Solus is in `/usr/share/defaults/etc/shells`.
+
+If you want to switch from *bash* to another shell in your terminal you have to use the command `chsh` to do that (alongside installing the proper package for your shell).
```
Comment (JoshStrobl) [inline configuration/changing-shell/en.md]: No information on installation of other shells.

File: configuration/changing-shell/en.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+The command is used with `sudo` rights to change the shell for your user session.
+
+Exemple:
```
Comment (JoshStrobl) [inline configuration/changing-shell/en.md]: Example, not Exemple.

**Comment by Bridouz** [comment]
Changes applied :)

**Pull request merged**